### PR TITLE
Add in SystemIdentifier blank node type and use of it in Entity (cc204/220)

### PR DESCRIPTION
**Pull request by ViktorHaag**
- Entity.otherIdentifiers is a list of SystemIdentifiers

- SystemIdentifier is a blank node type that can contain a system-local
  identifier for an entity

- System Identifier Types is a controlled vocabulary of system identifer types
  so we have a common vocabulary for identifying the general type of a
  system-local identifier.

- Note this PR covers adding in the `extensions` property for SystemIdentifier as per CC#220, but _not_ for the TextPositionSelector.

See also:
- [Caliper central issue #204](https://github.com/IMSGlobal/caliper-central/issues/204)
- [Cailper central issue #220](https://github.com/IMSGlobal/caliper-central/issues/220)

### CODE REVIEW

File: caliper-spec.md [hunk 1]
```diff
@@ -5616,7 +5619,46 @@ http://purl.imsglobal.org/caliper/TextPositionSelector
 }
 ```
 
-## <a name="roles"></a>Appendix E. Roles
+## <a name="systemIdentifiers"></a>Appendix E. System Identifiers
```
Comment (ViktorHaag) [inline caliper-spec.md:5622]: Note that these are not first-class Caliper Entities, they're a blank node type, like the TextPositionSelector, so they deserve their own Appendix section to document them.

File: caliper-spec.md [hunk 2]
```diff
@@ -293,6 +295,7 @@ Like an [Event](#event), an [Entity](#entity) is considered semi-structured data
 | description | string |  A string value comprising a brief, written representation of the [Entity](#entity). | Optional |
 | dateCreated | DateTime | An ISO 8601 date and time value expressed with millisecond precision that describes when the [Entity](#entity) was created.  The value MUST be expressed using the format YYYY-MM-DDTHH:mm:ss.SSSZ set to UTC with no offset specified. | Optional |
 | dateModified | DateTime | An ISO 8601 date and time value expressed with millisecond precision that describes when the [Entity](#entity) was last changed or updated.  The value MUST be expressed using the format YYYY-MM-DDTHH:mm:ss.SSSZ set to UTC with no offset specified. | Optional |
+| otherIdentifiers | Array | An ordered collection of [SystemIdentifier](#systemIdentifier) entities that represent other identifiers for parent entity known to the [Sensor](#sensor) application. Each array item MUST be expressed either an object or as a string corresponding to the item's [IRI](#iriDef). | Optional |
```
Comment (bracken) [inline caliper-spec.md]: "MUST be expressed either an object" is missing an "as"

**Comment by ViktorHaag** [comment]
> I don't know how much we do this in the spec in general, but I like to see a minimal example because that's what most people want, and then an example with all the things. As a spec consumer I love it when I can see both quickly. (not a strong opinion, just preference)

I can see two useful ways of doing that in the spec:
- Do it everywhere for all examples
- Do it in cases where the added example to show differences is usefully made around a particularly tricky concept or notable point

I think, either way, this could be done as a PR/effort entirely on its own outside other individual PRs like this?

(BTW -- adding useful non-normative, clarifying text like that would be in my opinion an excellent example of why we'd want to be able to republish a spec without revving it's official version number.)

**Comment by arwhyte** [comment]
I have added this content to the unitary v1p2 spec.

**Pull request merged**